cmd/faucet/fund.go

```diff
@@ -0,0 +1,106 @@
+package faucet
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/cmd/flags"
+	"github.com/pokt-network/poktroll/cmd/logger"
+)
+
+// fundURLFmt is the canonical fund URL format for a given denom and recipient address.
+// The placeholders are intended to be interpolated in the following order:
+//   - baseURL: Fully-qualified URL to the faucet server (e.g. https://shannon-testnet-grove-faucet.beta.poktroll.com)
+//   - denom: the denom to fund (e.g. upokt)
+//   - recipientAddress: the recipient address to fund (e.g. pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4)
+const fundURLFmt = "%s/%s/%s"
```

Comment:
Can you provide an example of a curl that makes it works? If my understand is correct.